stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift

```diff
@@ -0,0 +1,109 @@
+#if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
+import Darwin
+#else
+import Glibc
+#endif
+import CTensorFlow
+import CTensorFlowUnittest
+import TensorFlow
+import StdlibUnittest
+
+typealias CTFServer = OpaquePointer
+
+@usableFromInline
+func debugLog(_ message: @autoclosure () -> String,
+              file: StaticString = #file,
+              line: UInt = #line) {
+  if _RuntimeConfig.printsDebugLog {
+    print("[\(file):\(line)] \(message())")
+    // This helps dump more log before a crash.
+    fflush(stdout)
+  }
+}
+
+
+private class TestCluster {
+  var num_tasks:Int = 0
+  var servers: [CTFServer] = []
+  public var serverDef:String = ""
+
+  public init(num_tasks: Int) {
+    self.num_tasks = num_tasks
+    self.serverDef = startServers()
+  }
+
+  deinit {
+    // TODO: There is no clean way to shutdown GrpcServers yet.
+  }
+
+  private func getServerDef(tasks: String, task_index: Int) -> String {
```

Comment:
Please avoid using "set"/"get" in any Swift function name. And use camelCase.
```swift
private func serverDef(forTasks: String, taskIndex: Int) -> String {
```

Reply:
Done. getServerDef -> createServerDef. Similarly changed getTasks -> createTaskConfig